src/PowerShell/Microsoft.WinGet.Client.Cmdlets/Cmdlets/GetSettingCmdlet.cs

```diff
@@ -1,34 +1,35 @@
-﻿// -----------------------------------------------------------------------------
-// <copyright file="GetSettingsCmdlet.cs" company="Microsoft Corporation">
-//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
-// </copyright>
-// -----------------------------------------------------------------------------
-
-namespace Microsoft.WinGet.Client.Cmdlets.Cmdlets
-{
-    using System.Management.Automation;
-    using Microsoft.WinGet.Client.Common;
-    using Microsoft.WinGet.Client.Engine.Commands;
-
-    /// <summary>
-    /// Gets winget settings.
-    /// </summary>
-    [Cmdlet(VerbsCommon.Get, Constants.WinGetNouns.Settings)]
-    public sealed class GetSettingsCmdlet : PSCmdlet
-    {
-        /// <summary>
-        /// Gets or sets a value indicating whether to output a string or a hashtable.
-        /// </summary>
-        [Parameter(ValueFromPipelineByPropertyName = true)]
-        public SwitchParameter AsPlainText { get; set; }
-
-        /// <summary>
-        /// Get settings.
-        /// </summary>
-        protected override void ProcessRecord()
-        {
-            var command = new CliCommand(this);
-            command.GetSettings(this.AsPlainText.ToBool());
-        }
-    }
-}
+﻿// -----------------------------------------------------------------------------
+// <copyright file="GetSettingCmdlet.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------------
+
+namespace Microsoft.WinGet.Client.Cmdlets.Cmdlets
+{
+    using System.Management.Automation;
+    using Microsoft.WinGet.Client.Common;
+    using Microsoft.WinGet.Client.Engine.Commands;
+
+    /// <summary>
+    /// Gets winget settings.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, Constants.WinGetNouns.Setting)]
+    [Alias("gwgse", "Get-WinGetSetting")]
```

Comment:
```suggestion
    [Alias("gwgse", "Get-WinGetSettings")]
```
Could this alias duplication be interfering with the cmdlet export?  The tests are failing with:
```
##[error] [-] Get|Enable|Disable-WinGetSetting.Get-WinGetSetting 5ms (3ms|1ms)
##[error]  CommandNotFoundException: The term 'Get-WinGetSetting' is not recognized as a name of a cmdlet, function, script file, or executable program.
##[error]  Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
##[error]  at <ScriptBlock>, D:\a\1\s\src\PowerShell\tests\Microsoft.WinGet.Client.Tests.ps1:646
```
Which looks like the correct command to me.